frontend/test/pytest/test_static_arguments.py

```diff
@@ -221,6 +221,75 @@ def wrapper(x, c):
         captured = capsys.readouterr()
         assert captured.out.strip() == "Inside QNode: 0.5"
 
+    def test_static_argnames(self):
+        # pylint: disable=unused-argument, function-redefined
+        """Test static arguments specified by names"""
+
+        @qjit(static_argnames="y")
+        def f(x, y):
+            return
+
+        assert set(f.compile_options.static_argnums) == {1}
+
+        with pytest.raises(ValueError, match="qjitted function has invalid argname {yy}"):
```

Comment:
```suggestion
        with pytest.raises(ValueError, match="The `static_argnames` argument 'yy' in `qjit` is not a valid argument"):
```

If you plan to change the error message.